### PR TITLE
add update-distgit.py

### PR DESCRIPTION
**Pull request by ondrejbudai**
It currently lives in osbuild-composer's repository but it is actually quite
generic so let's move it here.

### CODE REVIEW

**Comment by ondrejbudai** [comment]
(force-pushed to fix pylint's complaints)

File: README.md [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
 
  - release.py
- - node_reservation.sh
\ No newline at end of file
+ - node_reservation.sh
+ - update-distgit.sh
+
+## `update-distgit.sh`
+
+A useful script for updating osbuild package downstream. When run in
```
Comment (ochosi) [inline README.md:12]:
```suggestion
A useful script for updating osbuild packages downstream. When run in
```

Alternatively you can also use the article 'the osbuild package downstream'.

**Review by ochosi — APPROVED** [review]
That overall makes a lot of sense and I've wanted to do this myself for a while. Thanks a bunch!

File: README.md (hunk 1 above)
Comment (ondrejbudai) [reply README.md]: Fixed, thanks!

**Review by ochosi — APPROVED** [review]
:heart:

**Pull request merged**